examples/seismic/tutorials/09_viscoelastic.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
Overall neeeds bit more explanation/comments/... a bit too brute force like that.
Maybe split the equations in different cells for velocities/stress/attenuation